### ISSUE TITLE
MDL component is not able load when requesting with ajax.

### ISSUE THREAD

**Issue by klarnol** [google/material-design-lite#5350]
I am working on spring boot project with jsp and meterial design lite. 

The jquery code looks like below.
 $.get("${pageContext.request.contextPath}/info/getform", function(data){
                $("#mainContent").html(data);
            }); 
    });
Material design lite code looks like below

<form action="#">
  <div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
    <input class="mdl-textfield__input" type="text" id="sample3">
    <label class="mdl-textfield__label" for="sample3">Text...</label>
  </div>
</form>
The behavior of display when performing request with form action attribute normal. But while request performing with jquery ajax request to backend generating same code. But text field is displaying abnormal.
It is very major issue while working with backend applications.
![behaviour of mdl component](https://user-images.githubusercontent.com/77790918/105378114-7cb01700-5c31-11eb-8acd-005ec26a7fb5.png)

**Comment by klarnol** [comment]
adding "window.componentHandler.upgradeDom();" like as shown below solved my issue

$.get("${pageContext.request.contextPath}/info/getform", function(data){
                $("#mainContent").html(data);
                window.componentHandler.upgradeDom(); 
            }); 
    });
